src/api/svg.ts

```diff
@@ -0,0 +1,537 @@
+import {
+  parse as parseHtml,
+  HTMLElement,
+  Attributes,
+} from 'node-html-better-parser';
+import { Color, colorString } from './colors';
+import { Degrees, degreesToRadians } from './rotations';
+import PDFPage from './PDFPage';
+import { PDFPageDrawSVGElementOptions } from './PDFPageOptions';
+import { LineCapStyle, LineJoinStyle } from './operators';
+
+interface Position {
+  x: number;
+  y: number;
+}
+
+interface Constraints {
+  width?: number;
+  height?: number;
+}
+
+type PDFPageDrawSVGElementOptionsRequireds = PDFPageDrawSVGElementOptions &
+  Position;
+
+interface SVGViewBox {
+  xMin: number;
+  yMin: number;
+  width: number;
+  height: number;
+}
+
+interface SVGSizeConverter {
```

Comment:
You need to convert x, y, width and height. X ant Y only won't be enough. Think about a scale transformation